<launch>

    <rosparam file ="$(find local_occupancy)/params/params.yaml" />
    <node name="occupancy_grid" pkg="local_occupancy" type="occupancy.py"/>
    <node type="rviz" name="rviz" pkg="rviz" args="-d $ default.rviz" />
</launch>